docs/standard/microservices-architecture/multi-container-microservice-net-applications/implement-api-gateways-with-ocelot.md

```diff
@@ -3,24 +3,24 @@ title: Implementing API Gateways with Ocelot
 description: Learn how to implement API Gateways with Ocelot and how to use Ocelot in a container-based environment.
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 07/03/2018
+ms.date: 10/02/2018
 ---
 # Implementing API Gateways with Ocelot
 
-The reference microservice application [eShopOnContainers](https://github.com/dotnet-architecture/eShopOnContainers) is using [Ocelot](https://github.com/ThreeMammals/Ocelot) because Ocelot is a simple and lightweight API Gateway that you can deploy anywhere along with your microservices/containers such as in any of the following environments used by eShopOnContainers.
+The reference microservice application [eShopOnContainers](https://github.com/dotnet-architecture/eShopOnContainers) is using [Ocelot](https://github.com/ThreeMammals/Ocelot), a simple and lightweight API Gateway that you can deploy anywhere along with your microservices/containers, such as in any of the following environments used by eShopOnContainers.
 
 - Docker host, in your local dev PC, on-premises or in the cloud.
 - Kubernetes cluster, on-premises or in managed cloud such as Azure Kubernetes Service (AKS).
 - Service Fabric cluster, on-premises or in the cloud.
 - Service Fabric mesh, as PaaS/Serverless in Azure.
 
-## Architect and design your API Gateways
+## Architecting and designing your API Gateways
```

Comment:
We try to avoid -ing forms of verbs. They do not localize well to many languages.

Reply:
Good to know! 

Should I remove -ing in all titles? as in:

- Designing and Developing Multi-Container and Microservice-Based .NET Applications
- Using a database server running as a container
- Creating a simple data-driven CRUD microservice
- Designing a simple CRUD microservice

There are actually 237 -ing titles out of 546 in the whole book ~42%

Reply:
I would remove -ing forms from any headers that you are modifying for other reasons. There are *so* many. I changed some in the first edition, and I think every update keeps lowering the number.

Reply:
Ok, I'm changing so many paragraphs, captions, titles and so on this time that I don't think it'll make any difference.

I'll keep an eye on this and try updating if the change sounds "good enough" to me, waiting for feedback from native English speakers. 😊